src/components/SafeListSidebar/index.tsx

```diff
@@ -114,7 +114,7 @@ export const SafeListSidebar = ({ children }: Props): ReactElement => {
             to={WELCOME_ADDRESS}
             variant="contained"
           >
-            + Add Safe
+            + Add account
```

Comment:
I think here we should be more careful. People can think they could add any account

Reply:
Yes, while not critical, this is a risk. However, it would become clear in the proceeding steps. We should aim for making it clear that the "account" in the context of Gnosis Safe is the Gnosis Safe account (vs. "owner" / "signer", which might be technically externally owned accounts but aren't referenced as accounts in Gnosis Safe). If we get user requests about this being confusing this would just indicate that we haven't explained this concept well enough during the onboarding.